src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
+    white,
+    red,
+    blue
+}
```

Comment:
First of all, the enum name should be single, not plural. `Color`, instead of `Colors`.

Then, enum values should be in upper snake case. See [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-forget-how-to-name-constants-according-to-style-guide-the-same-with-naming-of-enum-values).

Also, an enum class with no methods and no documentation on its constants may optionally be formatted as if it were an array initializer. See [here](https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes).

```suggestion
public enum Color { BLACK, WHITE, RED, BLUE }
```